modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/NotificationTemplateObjectActionExecutorImpl.java

```diff
@@ -126,8 +126,8 @@ private Map<String, Object> _getTermValues(
 				SimpleDateFormat simpleDateFormat = new SimpleDateFormat(
 					"yyyy-MM-dd");
 
-				Timestamp timestamp = (Timestamp)termValues.get(
-					objectField.getName());
+				Timestamp timestamp = new Timestamp(
+					(Long)termValues.get(objectField.getName()));
```

Comment:
Please, remember to consider this review [here](https://github.com/liferay-commerce/liferay-portal/pull/4838/files/08a59747ba023b850f13fee1fdf385cb5f18538c..506d15196c922900e451fb427f5cd9fef51d2b78#r1553912802). Thank you :smile: